src/Components/JournalEntry/JournalEntry.js

```diff
@@ -1,16 +1,18 @@
 import React from "react";
+import { DateTime } from 'luxon';
 import './JournalEntry.css';
 
 const JournalEntry = ({ journal }) => {
 
   const {date, feeling, activity, journalEntry} = journal;
+  const formattedDate = DateTime.fromFormat(date, 'yyyy-MM-dd')
 
   return (
     <section className='journal-boundary'>
       <div className='journal-entry'>
         <h2>Journal Entry</h2>
         <div className='log-stats'>
-          <h3>{date}</h3>
+          <h3>{formattedDate.toLocaleString(DateTime.DATE_MED)}</h3>
```

Comment:
heck yeah!

Reply:
also nice progression of headings! h1 for the header, then h2, and then some h3's. great work!